node/src/Router.ts

```diff
@@ -363,21 +365,24 @@ export class Router extends EnhancedEventEmitter<RouterEvents>
 			maxSctpMessageSize = 262144,
 			sctpSendBufferSize = 262144,
 			appData
-		}: WebRtcTransportOptions
-	): Promise<WebRtcTransport>
-	{
-		logger.debug('createWebRtcTransport()');
+		} = webRtcTransportOptions;
```

Comment:
🙁

I don't like this. We use fields directly rather than having a named object. This change is making a difference with no much reason.

Reply:
Older revision of the code didn't compile otherwise, changed back now.